libkbfs/rekey_queue.go

```diff
@@ -57,7 +61,9 @@ func (rkq *RekeyQueueStandard) Enqueue(id tlf.ID) <-chan error {
 			// spawn goroutine if needed
 			var ctx context.Context
 			ctx, rkq.cancel = context.WithCancel(context.Background())
-			go rkq.processRekeys(ctx, rkq.hasWorkCh)
+			for i := 0; i < numRekeyWorkers; i++ {
```

Comment:
`hasWorkCh` only has a buffer of size 1, and when `Enqueue` sends on it, there's a default case that does nothing.  I think there can be races where multiple Rekeys get enqueued, but only one worker thread is woken up to process them, because the other signals are dropped.  We should probably bump the channel size to `numRetryWorkers`, or change the queue into a proper channel itself.